SofaKernel/modules/SofaCore/src/sofa/core/MultiVecId.h

```diff
@@ -678,8 +606,16 @@ typedef TMultiVecId<V_MATDERIV, V_WRITE>     MultiMatrixDerivId;
 typedef TMultiVecId<V_ALL, V_READ>      ConstMultiVecId;
 typedef TMultiVecId<V_ALL, V_WRITE>          MultiVecId;
 
-} // namespace core
+//#if (SOFA_CORE_MULTIVECID_SKIP_EXTERN_TEMPLATE_DECLARATION == 1)
```

Comment:
Any reason keeping this commented code? If not, the macro declaration in the cpp file is also not needed.

Reply:
No :)